include/swift/AST/DiagnosticsSema.def

```diff
@@ -1908,6 +1908,10 @@ ERROR(inheritance_from_final_class,none,
 ERROR(inheritance_from_unspecialized_objc_generic_class,none,
       "inheritance from a generic Objective-C class %0 must bind "
       "type parameters of %0 to specific concrete types", (Identifier))
+ERROR(inheritance_from_class_with_missing_vtable_entries,none,
+      "cannot inherit from class %0 in Swift %1 mode because it has "
+      "overridable members that cannot be loaded",
```

Comment:
By this point that information has been thrown away, and even within deserialization it's hard to say. ("I can't load this because I couldn't load the thing it overrides because I couldn't find this particular type, probably because you changed language versions.") I can at least preserve the language version change, though.